### PR TITLE
CA-196244: Username/password made optional for SMBSR

### PR DESCRIPTION
**Pull request by chandrikas**
Signed-off-by: Chandrika Srinivasan <chandrika.srinivasan@citrix.com>

### CODE REVIEW

File: drivers/SMBSR.py [hunk 1]
```diff
@@ -197,6 +142,62 @@ def mount(self, mountpoint=None):
...
+    def getMountOptions(self):
+        """Creates option string based on parameters provided"""
+        options = ['sec=ntlm',
+                'cache=loose',
+                'vers=3.0',
+                'actimeo=0'
+        ]
+
+        if self.dconf.has_key('username') \
+                and (self.dconf.has_key('password') or self.dconf.has_key('password_secret')):
```
Comment (pritha-srivastava) [inline drivers/SMBSR.py]: Does the line exceed 80 chars?

**Comment by pritha-srivastava** [comment]
The changes look fine to me (except for a minor comment)

File: drivers/SMBSR.py (hunk 1 above)
Comment (chandrikas) [inline drivers/SMBSR.py]: It is indeed. I have made the required changes.

**Comment by pritha-srivastava** [comment]
Looks good.

**Pull request closed without merging**